### PR TITLE
Wip - do not merge - testing only

### PR DESCRIPTION
**Pull request by andi34**
# do not merge

Test open PR:
- api / lib adjustments https://github.com/PhotoboothProject/photobooth/pull/200
- nextcloud integration https://github.com/PhotoboothProject/photobooth/pull/208
- Adminpanel adjustments https://github.com/PhotoboothProject/photobooth/pull/209
- livechroma: cleanup, fix Post processing, renamed to "Chroma capture" as "livechroma" is misleading https://github.com/PhotoboothProject/photobooth/tree/fix/livechroma
-

### CODE REVIEW

**Comment by andi34** [comment]
Placed here to easily rebase onto latest dev to avoid
_Merge branch x into x_ commits

Also adjusted open PR to be based on the api/lib re-write

File: welcome/index.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+require_once '../lib/config.php';
+require_once '../lib/photobooth.php';
+
+if (!is_file('.skip_welcome')) {
+    touch('.skip_welcome');
+}
+
+$photobooth = new Photobooth();
+$URL = $photobooth->getUrl();
+$PHOTOBOOTH_HOME = getcwd();
```
Comment (andi34) [inline welcome/index.php]: Need to be adjusted after moving to welcome/

**Review by ledsi — APPROVED** [review]
What a huge upgrade. Looks nice to me, local testing worked quite nice and without any issues. Maybe some lines of code could get a bit eyecandy but this is not important for the merge.

**Comment by andi34** [comment]
Reboot and Shutdown don't work from Adminpanel. Can't find the reason. Can someone of you have also a look on it?

File: api/shellCommand.php [hunk 2]
```diff
@@ -3,8 +3,25 @@
 
 require_once '../lib/config.php';
 
+$Logger = new DataLogger(PHOTOBOOTH_LOG);
+$Logger->addLogData(['php' => basename($_SERVER['PHP_SELF'])]);
+
 $mode = $_POST['mode'];
 
+if (empty(mode)) {
```
Comment (andi34) [inline api/shellCommand.php]: $mode

File: admin/components/footer.admin.php [hunk 3]
```diff
@@ -1,7 +1,7 @@
     <div class="pageLoader w-full h-full fixed top-0 left-0 z-50 hidden place-items-center [&.isActive]:grid">
         <div class="w-full h-full left-0 top-0 z-10 absolute bg-black bg-opacity-60"></div>
         <div class="px-4 py-6 rounded-md bg-white shadow-md flex flex-col items-center justify-center relative z-20 text-center">
-            <?php echo getLoader("sm"); ?>
+            <?php echo getLoader("lg"); ?>
```
Comment (andi34) [inline admin/components/footer.admin.php:4]: need to be in here?

**Comment by andi34** [comment]
Also added changes to livechroma from https://github.com/PhotoboothProject/photobooth/tree/fix/livechroma

**Pull request closed without merging**